packages/core/getUserAvailability.ts

```diff
@@ -274,14 +281,28 @@ const _getUserAvailability = async function getUsersWorkingHoursLifeTheUniverseA
       ? await getBusyTimesFromLimits(
           bookingLimits,
           durationLimits,
-          dateFrom,
-          dateTo,
+          dateFrom.tz(timeZone),
```

Comment:
fixes existing bug that caused the following error when host and attendee had different timezones: 
![Screenshot 2024-09-17 at 4 08 21 PM](https://github.com/user-attachments/assets/43a2000c-d877-48de-8025-c1d6da825665)